src/ui/utils/timeline.ts

```diff
@@ -336,10 +336,10 @@ export function mergeSortedPointLists(
     if (destinationIndexPoint == null) {
       destination.push(sourcePoint);
       sourceIndex++;
-    } else if (sourcePoint.time === destinationIndexPoint.time) {
+    } else if (sourcePoint.point === destinationIndexPoint.point) {
       // Don't add duplicates.
       sourceIndex++;
-    } else if (sourcePoint.time < destinationIndexPoint.time) {
+    } else if (BigInt(sourcePoint.point) < BigInt(destinationIndexPoint.point)) {
```

Comment:
:question: We've got utils for comparing points already - should we use those here?

Reply:
Yeah, we have several of those and use them inconsistently. Originally they were introduced when we tried to avoid `BigInt` for Safari compatibility but that doesn't seem to be a concern anymore. :shrug: